proposals/load_geojson.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "load_geojson",
+    "summary": "Load GeoJSON data",
+    "description": "Loads GeoJSON data as defined by [RFC 7946](https://www.rfc-editor.org/rfc/rfc7946.html) into a vector data cube. Feature properties are preserved.",
+    "categories": [
+        "import",
+        "vector"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "The GeoJSON data. Can either be inline GeoJSON, a user-uploaded file, or point to a remote URL. The GeoJSON type `GeometryCollection` is not supported. Each geometry in the GeoJSON data results in a dimension label in the `geometries` dimension.",
+            "schema": [
+                {
+                    "title": "GeoJSON",
+                    "description": "GeoJSON (inline)",
+                    "type": "object",
+                    "subtype": "geojson"
+                },
+                {
+                    "title": "URL",
+                    "type": "string",
+                    "format": "uri",
+                    "subtype": "uri",
+                    "pattern": "^https?://"
+                },
+                {
+                    "title": "User-uploaded file",
+                    "type": "string",
+                    "subtype": "file-path",
+                    "pattern": "^[^\r\n\\:'\"]+$"
+                }
+            ]
+        },
+        {
+            "name": "properties",
+            "description": "One or more properties from the GeoJSON file to construct an additional dimension from. Only applies for GeoJSON Features and FeatureCollections. Missing values are generally set to no-data (`null`).\n\nDepending on the number of properties provided the process constructs and names the dimension differently:\n\n- Single property with scalar values: A single dimension label with the name of the property and a single value per geometry.\n- Single property of type array: The dimension labels correspond to the array indices. There are as many values and labels per geometry as there are for the largest array.\n- Multiple properties with scalar values: The dimension labels correspond to the property names. There are as many values and labels per geometry as there are properties provided here.\n\nDepending on the number of properties the dimension name is:\n\n- Single property: Name of the property\n- Multiple properties: `properties`.",
```

Comment:
>    Depending on the number of properties provided the process constructs and names the dimension differently:

What then follows is a discussion about the dimension labels (not dimension names).
Explanation of the dimension name is in another paragraph

Reply:
> Depending on the number of properties the dimension name is:\n\n- ... Multiple properties: properties.",

related to the discussion about dim name geometries/geometry (https://github.com/Open-EO/openeo-api/issues/479 settled on singular geometry), this should be singular "property" as well, right?

Reply:
1) True, improved.
2) Not sure, it was just about recommended names. There we opted for geometry due to the Python ecosystem, but here we are clearly following GeoJSON / SF terminology.